qa/rpc-tests/sc_cert_memcleanup_split.py

```diff
@@ -250,7 +235,8 @@ def run_test(self):
 
         #============================================================================================
         mark_logs("\nJoining network", self.nodes, DEBUG_MODE)
-        self.join_network()
+        self.join_network(2)
+        time.sleep(2)
```

Comment:
That's more a memo for me, actually, but I would like to investigate what is the purpose of this sleep and, eventually, if there could be a more reliable time-independent method

Reply:
There was a sleep in the join_network method of that class and without it the test tend to fail.